hw_1/vipcls data.R

```diff
@@ -14,18 +14,15 @@
 ##
 ## ---------------------------
 
-## Packages
+## Install Packages
 library(tidyverse)
 
-## Import data
+## Create a data frame by reading data from "vipcls.csv"
 vipcls <- read.csv("vipcls.csv")
 
 ## Change WD to Epi 510
-j_wd = "~/Repositories/epi_510/hw_1"
-setwd(j_wd)
-
-## Create a dataframe: 
-vipcls <- data.frame(vipcls)
+wd = "~/Repositories/epi_510/hw_1"
+setwd(wd)
 
```

Comment:
Setting wd this way will ensure that it works for both of us! Since we both have things organized the same way. :)